### ISSUE TITLE
While creating customer, Loan etc.  Header of stepper is not interactive

### ISSUE THREAD

**Issue by therajanmaurya** [therajanmaurya/android-client-2.0#117]
**Summary:** 

The header of stepper is not interactive e.g. when the user wants to jump to the first step instead of using the back button. 

I made this intentionally. I need to do some research that how our code can manage to enable to do it.

**Comment by AkshGautam** [comment]
@therajanmaurya In this enhancement will the user be permitted to jump from step 1 to step 3 even if the user has not filled the details of step 1 and 2? I would like to think about the solution and possible work on this.

**Comment by therajanmaurya** [comment]
No, the user can't jump to step 3 if he did not fill the form 1-2. The user can jump to the forms which form he filled.
